### ISSUE TITLE
Cholestane-3β,5α,6β-triol high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11239]
**Preferred term label:**
Cholestane-3β,5α,6β-triol high in blood

**Synonyms**
C-triol

**Definition (free text, please give PubMed ID)**
The compound, but not its elevation, is in https://uts.nlm.nih.gov/uts/umls/concept/C5144645

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Niemann Pick C
Wolman disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001353
- label: Elevated circulating C-triol concentration
- parents: [Abnormal circulating lipid concentration (HP:0003119)](https://hpo.jax.org/app/browse/term/HP:0003119)
- pmid: PMID:32178982
- def: The concentration of cholestane-3beta, 5alpha, 6beta-triol (C-triol) in the blood circulation is above the upper limit of normal. C-triol is a diagnostic biomarker for Niemann-Pick disease type C.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11239
- synonyms: Elevated circulating cholestane-3beta,5alpha,6beta-triol concentration(NONE)
